bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -60,7 +60,7 @@ spec:
           name: etc-openvswitch
         - mountPath: /var/lib/openvswitch/
           name: var-lib-openvswitch
-        - mountPath: /run/openvswitch/
```

Comment:
/var/run should be a symlink to /run -- is this needed?

Reply:
Indeed it is not. Thought it was when I tested, but turns out there was another issue.